### PR TITLE
Feature/simplecov

### PR DESCRIPTION
**Pull request by stasfly**
Add simplecov.

### CODE REVIEW

**Review by mmrshk — COMMENTED** [review]
Coverage folder add to .gitignore please

File: db/schema.rb [hunk 1]
```diff
@@ -45,6 +45,74 @@
     t.index ["ancestry"], name: "index_documents_on_ancestry"
   end
 
+  create_table "lit_incomming_localizations", id: :serial, force: :cascade do |t|
```
Comment (mmrshk) [inline db/schema.rb:48]: why your schema was changed?

File: bin/rails [hunk 2]
```diff
@@ -4,3 +4,8 @@
 APP_PATH = File.expand_path('../config/application', __dir__)
 require_relative '../config/boot'
 require 'rails/commands'
+if ENV['RAILS_ENV'] == 'test'
+  require 'simplecov'
+  SimpleCov.start 'rails'
+  puts "required simplecov"
+end
```
Comment (mmrshk) [inline bin/rails:11]: add line

File: db/schema.rb (hunk 1 above)
Comment (stasfly) [reply db/schema.rb:48]: Every time I do git pull I also  run 'bundle install' and 'rails db:migrate'.
Probably the last one made some changes to schema.rb.
In any event I did no changes in the db/schema.rb explicitly.

File: bin/rails (hunk 2 above)
Comment (stasfly) [reply bin/rails:11]: Added.

**Review by mmrshk — APPROVED** [review]
lgtm

**Pull request merged**